src/options.mjs

```diff
@@ -50,6 +52,9 @@ export const defaultOptions = {
 	// APP1 - IFD1 - Size and other information about embeded thumbnail.
 	thumbnail: false,
 
+	// XMP Parser function
+	xmpParser: XMPParser.read,
```

Comment:
I now parse the XMP string by default, but you could specify another parser function.

Reply:
This is to not divert too much from upstream.